tests/sdk/nodejs/preview-auth/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016-2023, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as k8s from "@pulumi/kubernetes";
+
+// Create provider with SSA enabled.
+const provider = new k8s.Provider("k8s", {enableServerSideApply: true});
+
+new k8s.core.v1.ConfigMap("test", {
+    metadata: {
+        name: "foo",
```

Comment:
This resource does not actually get applied to the cluster since the service account lacks permissions to actually create the resource. [L94-96 in preview_test.go](https://github.com/pulumi/pulumi-kubernetes/pull/2522/files#diff-6f7cec5fb196b479ea7d332323fb587b0b74fc300c7810f55c30aa65495c6fc8R94-R96) asserts this to be true.

But I'll update the resource to use a randomly generated name in case this assumption fails in the future!